lib/msal-core/src/Utils.ts

```diff
@@ -538,6 +538,24 @@ export class Utils {
     return url;
   }
 
+  /**
+   * @hidden
+   * @ignore
+   *
+   * Returns the anchor part(#) of the URL
+   */
+  static getHashFromUrl(urlStringOrFragment: string): string {
+    let hash = urlStringOrFragment;
+    const hashIndex1 = urlStringOrFragment.indexOf("#");
+    const hashIndex2 = urlStringOrFragment.indexOf("#/");
+    if (hashIndex2 > -1) {
+      hash = urlStringOrFragment.substring(hashIndex2 + 2);
+    } else if (hashIndex1 > -1) {
+      hash = urlStringOrFragment.substring(hashIndex1 + 1);
+    }
+    return hash;
```

Comment:
Instead of reassigning, you can early return:

```js
if (hashIndex2 > -1) {
  return urlStringOrFragment.substring(hashIndex2 + 2);
}

if (hashIndex1 > -1) {
   return urlStringOrFragment.substring(hashIndex1 + 1);
}

return hash;
```

Reply:
done